config/routes.rb

```diff
@@ -4,9 +4,11 @@
   devise_for :buyers
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'pages#home'
+  get 'artists/:id' => 'artists#show'
+  post 'commissions/create' => 'commissions#create'
```

Comment:
As mentioned before, post request should have the route `post 'commissions/' => 'commissions#create'`. Also, I believe that this controller should exist under the `api` namespace. This means that we want to organize it under the `api/` folder within the controllers because CRUD methods seem more like api methods than independent methods outside of the api folder.

Reply:
Check out how the `Works` table was created (and routed to) for a good example.